src/common/utility.ts

```diff
@@ -284,9 +284,16 @@ export class Utility {
         return undefined;
     }
 
-    public static async loadEnv(envFilePath: string) {
+    public static async loadEnv(filePath: string) {
         try {
-            if (await fse.pathExists(envFilePath)) {
+            const workspaceFolder = vscode.workspace.getWorkspaceFolder(vscode.Uri.file(filePath));

```

Comment:
check multi-root nested scenario. And how about use the vscode.uri selected by user from ui to get the workspacefolder?